src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorOperation.cs

```diff
@@ -191,8 +191,8 @@ ref destination
             xRentedBuffer.Dispose();
         }
 
-        public static void Invoke<TOperation, TArg, TResult>(in ReadOnlyTensorSpan<TArg> x, in ReadOnlyTensorSpan<TArg> y, in TensorSpan<TResult> destination)
-            where TOperation : TensorOperation.IBinaryOperation_Tensor_Tensor<TArg, TResult>
+        public static void Invoke<TOperation, T1, T2, TResult>(in ReadOnlyTensorSpan<T1> x, in ReadOnlyTensorSpan<T2> y, in TensorSpan<TResult> destination)
+            where TOperation : TensorOperation.IBinaryOperation_Tensor_Tensor<T1, T2, TResult>
         {
```

Comment:
nit:
This looks ok but I am wondering if we better name the generic types to `TArg1` and `TArg2` would be better?

Reply:
I'm happy to make the switch, but we do have T1, T2 in many other places in that file. Think I should just switch them all at the same time?

Reply:
It is up to you if you want to have a consistent code in the whole class. I see we chose the name `TArg` for a good reason I guess instead of `T`. That is why I suggested to rename `T`s. I'll leave it to you to decide if you care about consistency and naming.